README.markdown

```diff
@@ -34,11 +34,12 @@ Writing Objective-C? Check out our [Objective-C Style Guide](https://github.com/
 
 ## Naming
 
-Use descriptive names with camel case for classes, methods, variables, etc. Class names should be capitalized, while method names and variables should start with a lower case letter.
+Use descriptive names with camel case for classes, methods, variables, etc. Class names should be capitalized, while method names and variables should start with a lower case letter. Global constants, however, should have a project prefix.
 
 **Preferred:**
 
 ```swift
+let XYZSomeGlobalConstant = 123.456
```

Comment:
I see. I'm completely ignorant when it comes to namespacing this kind of stuff. The standard library doesn't have any examples, and browsing the web a bit isn't that helpful.

I think enforcing global variable namespacing through structs (rather than clases) is a very elegant solution, but seems like a bit more of work.

Usage:

`Module.Struct.Constant` vs `ABCConstant`.

Usage wise, I would vote in favor of first option. Implementation wise, I would lean on the second.

I'll give a neutral vote: :+1: :-1: